README.md

```diff
@@ -118,38 +118,6 @@ calling the `emogrify` method:
 * `$emogrifier->addExcludedSelector(string $selector)` - Keeps elements from
   being affected by emogrification.
 
-## Installing with Composer
-
-Download the [`composer.phar`](https://getcomposer.org/composer.phar) locally
-or install [Composer](https://getcomposer.org/) globally:
-
-```bash
-curl -s https://getcomposer.org/installer | php
-```
-
-Run the following command for a local installation:
-
-```bash
-php composer.phar require pelago/emogrifier:^2.2
-```
-
-Or for a global installation, run the following command:
-
-```bash
-composer require pelago/emogrifier:^2.2
-```
-
-You can also add follow lines to your `composer.json` and run the
-`composer update` command:
-
-```json
-"require": {
-  "pelago/emogrifier": "^2.2"
-}
-```
-
-See https://getcomposer.org/ for more information and documentation.
```

Comment:
I agree we don't need to document how to use Composer.  But is it still worth retaining the link to getcomposer.org – moving it to the end of the "Installation" section, and perhaps adding "… on Composer" to the end of the sentence?

Reply:
Agreed. Done.